src/org/labkey/test/tests/issues/IssuesTest.java

```diff
@@ -820,14 +819,28 @@ public void defaultAssignedToTest()
         // taking care of some clean-up while here for the test.
         _userHelper.deleteUsers(true, deletedUser, user);
 
+        //No NPE when the default user is selected.
         clickProject(getProjectName());
         waitAndClickAndWait(Locator.linkContainingText(ISSUE_SUMMARY_WEBPART_NAME));
         clickButton("New Issue");
-        // NPE
-        //clickButton("Cancel");
+        clickButton("Cancel");
 
-        // TODO: extend test to check validate full user selection list based on group selection...
         // TODO: compare user dropdown list between admin and new issues page
```

Comment:
is this TODO still needed or is this covered in the check at line 843? maybe I am misreading it and there is more to do.